src/js/envelope.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  EventEnvelope,
+  EventEnvelopeHeaders,
+  UserFeedback,
+  UserFeedbackItem,
+} from '@sentry/types';
+import { createEnvelope } from '@sentry/utils';
+
+/**
+ * Creates an envelope from a user feedback.
+ */
+export function createUserFeedbackEnvelope(
+  feedback: UserFeedback
+): EventEnvelope {
+  const headers: EventEnvelopeHeaders = {
+    event_id: feedback.event_id, // not sure feedback.event_id is correct
+    sent_at: new Date().toISOString(),
+  };
```

Comment:
The header should be enriched as such:
`let env = createEventEnvelope(event, this._dsn, this._options._metadata, this._options.tunnel);`
so the envelope header has all the needed metadata.